modules/social_features/social_event/modules/social_event_type/social_event_type.module

```diff
@@ -71,9 +71,9 @@ function social_event_type_preprocess_node(array &$variables) {
     $event_type = $node->get('field_event_type');
     $event_type_entities = $event_type->referencedEntities();
     if (count($event_type_entities) === 1) {
-      /** @var \Drupal\node\Entity\Node $event */
+      /** @var \Drupal\taxonomy\TermInterface $event */
```

Comment:
@dmytriikaiun We usually use the variable name `$event` for Node, and I think this is where the issue comes from.
So maybe to avoid this in the future better to rename `$event` to something like `$event_term` or `$event_type`?